docs/prometheus.md

```diff
@@ -0,0 +1,21 @@
+## syzkaller metrics
+
+syz-manager metrics are exposed at the URI : http://<syz-manager-http-uri>/metrics
+The prefix for metrics of importance is `syz_`
+
+These metrics are exported in prometheus format and can be ingested using following prometheus client configuration:
+```
+scrape_configs:
+- job_name: syzkaller
+  scrape_interval: 10s
+  static_configs:
+  - targets:
+    - localhost:56741
+```
+
+Values are reset to zero at syz-manager restart and only reflect for current execution of syz-manager.
```

Comment:
Should we expose deltas then?

Reply:
At the moment, deltas are not going to be particularly useful to tweak system configuration for fuzzing. Prometheus/Grafana has the ability to retain time series information and we can track the value of the metrics as time-series information object.

Reply:
> At the moment, deltas are not going to be particularly useful to tweak system configuration for fuzzing.

Why?
Prometheus sure should be able to build total sum from deltas, no? And deltas will work across restarts. Or should we expose both?
Let's say I want to count total number of test programs executed. Currently I can't because with every restart the count goes to 0...